web/crux-ui/src/components/projects/versions/deployments/deployment-status-tag.tsx

```diff
@@ -36,7 +36,7 @@ const statusToTextColor = (status: DeploymentStatus) => {
   }
 }
 
-const deploymentStatusTranslation = (status: DeploymentStatus): string => {
+export const deploymentStatusTranslation = (status: DeploymentStatus | 'all'): string => {
```

Comment:
What's the `all` case for?

Reply:
The type 'all' is not assignable to type if I'm missing from here. If there is a better approach, I'm happy to improve this. In general, I don't agree with the `deploymentStatusTranslation` function because it creates more depth complexity instead of using just the two common translations more times, but @m8vago implemented this and, as he said, I have to use it for now.

Reply:
Right now as a quick fix, I believe it's fine.
You just have add the `'all'` case to the switch statement.